apps/admin-portal/src/pages/identity-provider-template.tsx

```diff
@@ -38,8 +38,9 @@ import { addAlert } from "@wso2is/core/store";
 import { AlertLevels } from "@wso2is/core/models";
 import { AppState } from "../store";
 import { setAvailableAuthenticatorsMeta } from "../store/actions/identity-provider";
-import { SupportedServicesInterface } from "../models/identity-provider";
+import { SupportedServicesInterface } from "../models";
 import { IdPCapabilityIcons } from "../configs";
+import { ExpertModeTemplate } from "../components/identity-providers/meta/templates";
```

Comment:
Shall we export this form `components` index.

Reply:
How about now?